apps/dashboard/src/components/workflow-editor/workflow-canvas.tsx

```diff
@@ -59,7 +59,7 @@ const mapStepToNode = (
 ): Node<NodeData, keyof typeof nodeTypes> => {
   let content = '';
   if (step.type === StepTypeEnum.DELAY) {
-    content = `Wait to send ~ 30 minutes`;
+    content = `Delay sending for a specified duration`;
```

Comment:
Had to change this to something that doesnt specify the time.

If we want to make this react to the actual delay duration we would need `controlValues` in the workflow.steps response.